docs/src/pages/premium-themes/gmail/modules/components/search.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/styles';
+import InputBase from '@material-ui/core/InputBase';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import SearchIcon from '@material-ui/icons/Search';
+
+const useStyles = makeStyles(theme => ({
+  search: {
+    position: 'relative',
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: fade(theme.palette.common.black, 0.15),
+    '&:hover': {
+      backgroundColor: fade(theme.palette.common.black, 0.25),
+    },
+    marginRight: theme.spacing.unit * 2,
+    marginLeft: 0,
+    width: '100%',
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: theme.spacing.unit * 3,
+      width: '100%',
+    },
+  },
+  searchIcon: {
+    width: theme.spacing.unit * 9,
```

Comment:
This pattern is deprecated now.

```suggestion
    width: theme.spacing(9),
```

Reply:
I kinda missed that, thanks for pointing it out